CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+Contributing Translations to Search.gov
+=========================
+
+Note: As of November 2021, translations management has moved from the [GSA/Punchcard](https://github.com/GSA/punchcard/) repository to the [GSA/search-gov](https://github.com/GSA/search-gov) repository.
+
+Our language translations are the work of multiple contributors. You're encouraged to submit [pull requests](https://github.com/GSA/search-gov/pulls), [propose features and discuss issues](https://github.com/GSA/search-gov/issues).
+
+
+#### Contributing to localization files
+
+These files are in the YAML format.
+
+You can edit existing files or create new files directly via the [Github web interface](https://github.com/GSA/search-gov/tree/master/config/locales). Or, you can use Git from the command line (see below). Either way, follow these steps to create a new localization file:
+
+1. Start with `non_es_en_template.yml` and copy it into your new locale file. The reason you are using `non_es_en_template.yml` and not `en.yml` or `es.yml` is because the English and Spanish locale files contain many translations that are not used in other locales.
+1. Change the two letter locale in line 1 of your new file from `non_es_en_template` to match the locale of the filename.
+1. Be sure to follow the instructions for date-related fields like `cdr_format`, `date_format`, and `slashes`. Case is important!
+1. Be careful with preserving any opening/closing quotes around strings.
+1. Once you have the file ready, copy/paste it into a [YAML validator](http://www.yamllint.com) to ensure what you have is valid YAML.
+1. Commit the change in Github to a feature branch.
+
+#### Fork the Project
+
+If you don't want to use the [Github web interface](https://github.com/GSA/search-gov/tree/master/config/locales), you can use the command line tools to fork the [project on Github](https://github.com/GSA/search-gov) and check out your copy.
+
+```
+git clone https://github.com/contributor/search-gov.git
```

Comment:
When I was reading through this, I didn't immediately realize that I should replace "contributor" with my GH username. Is this a common convention though? Or is there additional styling we should add to make it more clear (ie. italicize it?)

Reply:
@amanifarooque , I agree that that should be clearer. However, we could also just point folks to the official GH guidance on forking & cloning: https://docs.github.com/en/get-started/quickstart/fork-a-repo. I took a quick look at the `CONTRIBUTING.md` for Rubocop (just because they're a huge, mature project with lots of GH goodness), and they just assume folks know how to fork a repo: https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#pull-requests